src/common/3Dpoint.hpp

```diff
@@ -0,0 +1,13 @@
+#ifndef geometry_H
+#define geometry_H
+
+
+class Geometry
+{
```

Comment:
you need to declare the constructor first and then the variable. Also the indent is usually 1 for public private protected and 2 for the variables. Don't use tabulation.

```cpp
class Geometry {
 public:
  Geometry(float, float, float);

  float x;
  float y;
  float z;
};
```